index.js

```diff
@@ -41,6 +71,8 @@ function run() {
   });
 
   if (changelog_was_updated) {
+    echo('...verifying changelog');
+    verifyChangelog();
```

Comment:
I don't necessarily like the idea of a script like this committing for me, BUT, it is nice to have the commit message automatically generated. So I think I will implement the try-catch.